### PR TITLE
Revise apk package handling

### PR DESCRIPTION
**Pull request by jackgr**
/cc @runseb

### CODE REVIEW

File: rootfs/expandybird/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@ FROM alpine:3.3
 COPY . /
 
 # install common packages
-RUN apk add --update-cache curl python
+RUN apk add --no-cache curl python
 
 WORKDIR /opt
```
Comment (runseb) [inline rootfs/expandybird/Dockerfile]: don't need this I think

File: rootfs/expandybird/Dockerfile [hunk 2]
```diff
@@ -17,7 +17,7 @@ FROM alpine:3.3
 COPY . /
 
 # install common packages
-RUN apk add --update-cache curl python
+RUN apk add --no-cache curl python
```
Comment (runseb) [inline rootfs/expandybird/Dockerfile]: `RUN apk add --no-cache curl python py-pip`

File: rootfs/expandybird/Dockerfile [hunk 3]
```diff
@@ -17,22 +17,13 @@ FROM alpine:3.3
 COPY . /
 
 # install common packages
-RUN apk add --update-cache curl python
+RUN apk add --no-cache curl python py-pip
 
 WORKDIR /opt
```
Comment (runseb) [inline rootfs/expandybird/Dockerfile]: don't think you need this

**Comment by runseb** [comment]
I think you could remove the `WORKDIR` line aside of that:

+1 LGTM

**Comment by jackgr** [comment]
Done. Thanks.

**Comment by jackgr** [comment]
#474

**Pull request merged**